generate.coffee

```diff
@@ -86,8 +87,8 @@ generateAddresses = (total) ->
     address =
       entity_id: index
       city: chance.city()
-      state: chance.state()
-      country: chance.country({full:true})
+      region: regions.region_id[chance.state()]
```

Comment:
I'm going to switch the generator to be US only and region to be a random state for now.